src/SFA.DAS.RequestApprenticeTraining.Application.UnitTests/Commands/WhenValidatingSubmitProviderResponseCommand.cs

```diff
@@ -89,5 +89,21 @@ public void Validate_EmployerRequests_Null_ShouldHaveValidationError()
             var result = _sut.TestValidate(model);
             result.ShouldHaveValidationErrorFor(x => x.EmployerRequestIds);
         }
+
+        [Test]
+        public void Validate_ContactName_NotEmpty_ShouldHaveNoValidationError()
+        {
+            var model = new SubmitProviderResponseCommand { ContactName = "Firstname Surame" };
```

Comment:
I guess Surame is a typo - and not the former capital of the Kebbi Kingdom - :)

Reply:
fixed